src/test/TestUtils.js

```diff
@@ -0,0 +1,80 @@
+/*
+Example usage:
+    var x1 = new UnitTest(()=>{return assertNotEqualsPrimitives(5,1)}, "5!=1 Test")
+    var x2 = new UnitTest(()=>{return assertTrue(false)}, "Wrong Test")
+    var suite = new TestSuite([x1, x2]);
+    suite.run()
+ */
+
+
+function assertEqualsObjects(expected, actual){
+    return expected.equals(actual)
+}
+
+function assertNotEqualsObjects(expected, actual){
+    return !expected.equals(actual)
+}
+
+function assertEqualsPrimitives(expected, actual){
+    return expected === actual
+}
+
+function assertNotEqualsPrimitives(expected, actual){
+    return !(expected === actual)
+}
+
+// message - expected error message, fun - function reference, args - arguments of fun as list
+function assertThrows(message, fun, args){
+    try {
+        for (var i = 0; i < args.length; i++)
+            fun = fun.apply(args[i]);
+        return false
+    } catch(error){
+        return message === error.message;
+    }
+}
+
+function assertFalse(condition){
+    return !condition
+}
+
+function assertTrue(condition){
+    return condition
+}
+
+function fail(message){
+    throw new Error("Test failed: " + message)
+}
+
+//fun should be arg-free function returning boolean (passed or not)
```

Comment:
Rozumiem, że każdy test to funkcja zwracająca boolean - brzmi sensownie. Zastanawiam się jednak co jeśli testy rozdzielimy do kilku plików. Trzeba będzie każdy odpalać osobno, co nie jest za bardzo wygodne. Rozważyłbym jeszcze taką możliwość

Reply:
Ok, już jest na tym branchu

Reply:
Tu właściwie się zastanawiam, czy testy nie powinny być odpalane z poziomu MuseScore'a?
Importy się nie kompilują w IDEI